config/sync/seckit.settings.yml

```diff
@@ -0,0 +1,55 @@
+seckit_xss:
```

Comment:
Snuck that in here eh?

Reply:
it came out during an export so i figured what the heck.